horace_core/sqw/file_io/@sqw_file_interface/sqw_file_interface.m

```diff
@@ -135,7 +135,8 @@
     methods(Abstract)
         % retrieve different parts of sqw data
         %------------------------------------------------------------------
-        [header,pos]= get_header(obj,varargin);
+        [header,pos,runid_map]...
```

Comment:
Why split this line? It's not very long.

Reply:
though they were aligned on `=`, but alignment is not kept by Matlab formatter